aries-site/src/utils/search.js

```diff
@@ -0,0 +1,38 @@
+import { structure } from '../data';
+
+export const formatName = name => {
+  return name
+    .split(' ')
+    .join('-')
+    .toLowerCase();
+};
+
+const getTopicName = itemName => {
```

Comment:
Glad you added these utility functions. I had started something similar, but will refactor to use this. Thank you.